### PR TITLE
Enable tests on 3.8+

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

**Comment by Dreamsorcerer** [comment]
@panagiks Not sure if you've got any ideas on this last warning? Seems there is a socket not getting closed, but not sure where. The port number is 6379, which appears to be for the redis server.

**Comment by asvetlov** [comment]
See https://github.com/aio-libs/aioredis-py/issues/1103 for woraround:  
```
         await client.close()
+        await client.connection_pool.disconnect()
```

File: tests/conftest.py [hunk 1]
```diff
@@ -17,6 +17,10 @@
     from typing_extensions import TypedDict
 
 
+# Testing...
+aioredis.Redis.__del__ = lambda *args: None  # type: ignore  # noqa
```
[the remark this answers is not in the document]
Comment (Dreamsorcerer) [reply tests/conftest.py:21]: Sorry, forgot to document this. This is a bug in aioredis. The session is closed properly, but `__del__` creates a task without awaiting on it, causing an error. https://github.com/aio-libs/aioredis-py/issues/1115

This line can be removed once that code is fixed in aioredis. The rest of the code is totally correct.

**Comment by Dreamsorcerer** [comment]
Finally! :P

**Comment by asvetlov** [comment]
My congratulations!

**Pull request merged**